src/lib/network/network_autoyast.rb

```diff
@@ -144,7 +144,7 @@ def activate_s390_devices(section = nil)
     def configure_hosts(write: false)
       log.info("NetworkAutoYast: Hosts configuration")
 
-      unless ay_current_profile.key? "host"
+      if ay_current_profile.empty?
```

Comment:
This rewrite is strange, originally it was call only if "host" section was present, and now if the profile is empty? at least is what the call and variable name suggest..